src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -9,61 +9,109 @@ import { connect } from 'react-redux';
 
 import FontAwesomeStyles from 'font-awesome/css/font-awesome.min.css';
 import { assetActions } from '../../data/constants/actionTypes';
-import { clearAssetsStatus, deleteAsset } from '../../data/actions/assets';
+import { clearAssetsStatus, deleteAsset, sortUpdate } from '../../data/actions/assets';
 
 export class AssetsTable extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      displayAssetsList: [],
       modalOpen: false,
       statusAlertOpen: false,
       assetToDelete: {},
+      deletedAsset: {},
+      deletedAssetIndex: null,
       elementToFocusOnModalClose: {},
-      statusAlertRef: {},
+      columnSortState: {
+        display_name: 'none',
+        content_type: 'none',
+        date_added: 'desc',
+      },
     };
 
+    this.columns = {
```

Comment:
I've moved the columns from the render up into an instance variable and added the columnSortable key to specify which columns are sortable.